frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsListComponent.tsx

```diff
@@ -20,28 +20,29 @@ interface FieldListProps {
   farmDetails: FarmDetailsInterface;
 }
 
-const FieldsListComponent: React.FC<FieldListProps> = ({ farmDetails }) => {
+const FieldsListComponent: FC<FieldListProps> = ({ farmDetails }) => {
   const fieldCount = farmDetails.Fields.length;
   // Will put this here for the meantime until I get insights from Product Owner
   const highValue = '25';
   const highPH = '4';
+
   return (
     <StyledFieldInfoList>
-      {farmDetails.Fields.map((fields: FieldDetailInterface) => (
-        <div key={fields.FieldName + fields.Area + fields.Comment}>
+      {farmDetails.Fields.map((field: FieldDetailInterface) => (
+        <div key={field.FieldName + field.Area + field.Comment}>
```

Comment:
Changed 'fields' to singular 'field' since in the map, we only access a singular field.